OPaL won't compile on flip

```
flip2 ~/CS467/OPaL 1016$ git branch
* main
flip2 ~/CS467/OPaL 1017$ git status
# On branch main
nothing to commit, working directory clean
flip2 ~/CS467/OPaL 1018$ make all_tests
# Delete binaries, output, temporary & report files 
rm -fv build/* output/* tmp/* report/*
mkdir -pv build tmp log report doc output
mkdir: created directory ‘doc’
gcc -g -O0 -fPIC -c -Wall src/opal.c -o build/opal.o
In file included from src/opal.c:5:0:
src/../include/opal.h:3:29: fatal error: bits/types/FILE.h: No such file or directory
 #include <bits/types/FILE.h>
                             ^
compilation terminated.
make: *** [libopal] Error 1
flip2 ~/CS467/OPaL 1019$ 
```

```
flip2 ~/CS467/OPaL 1022$ make all_tests
# Delete binaries, output, temporary & report files 
rm -fv build/* output/* tmp/* report/*
mkdir -pv build tmp log report doc output
gcc -g -O0 -fPIC -c -Wall src/opal.c -o build/opal.o
src/opal.c: In function ‘get_identifier_lexeme’:
src/opal.c:990:3: error: ‘for’ loop initial declarations are only allowed in C99 mode
   for (int i = 0; i < (sizeof(keyword_arr) / sizeof(keyword_arr[0])); i++)
   ^
src/opal.c:990:3: note: use option -std=c99 or -std=gnu99 to compile your code
make: *** [libopal] Error 1
flip2 ~/CS467/OPaL 1023$ 
```

--- a/include/opal.h
+++ b/include/opal.h
@@ -1,11 +1,11 @@
 /// @file opal.h
 
-#include <bits/types/FILE.h>
-#include <stdbool.h>            /* boolean datatypes */
-#include <stddef.h>
-
 #ifndef OPAL_H_
 #define OPAL_H_
+
+#include <stdio.h>
+#include <stdbool.h>            /* boolean datatypes */
+#include <stddef.h>
 
 /// __VERSION_NUM for program
 #ifndef __VERSION_NUM

--- a/src/alex.c
+++ b/src/alex.c
@@ -2,7 +2,6 @@
 
 #include <argp.h>
 #include <assert.h>
-#include <bits/types/FILE.h>
 #include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>     /* fclose */

--- a/src/astro.c
+++ b/src/astro.c
@@ -2,7 +2,6 @@
 
 #include <argp.h>
 #include <assert.h>
-#include <bits/types/FILE.h>
 #include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>     /* fclose */

--- a/src/opal.c
+++ b/src/opal.c
@@ -327,7 +327,7 @@ opal_error (short exit_code, char *log_msg, int fmt_option, char *fmt, ...)
     {
         logger (ERROR, log_msg);
     }
-    fprintf(stderr, log_msg);
+    fprintf(stderr, "%s", log_msg);
     return opal_exit(exit_code);
 }
 
@@ -987,7 +987,8 @@ get_identifier_lexeme (int char_line, int char_col)
     }
 
   /// Determine if string is a reserved keyword
-  for (int i = 0; i < (sizeof(keyword_arr) / sizeof(keyword_arr[0])); i++)
+  int i = 0;
+  for (i = 0; i < (sizeof(keyword_arr) / sizeof(keyword_arr[0])); i++)
     {
       if (strcmp (identifier_str, keyword_arr[i].str) == 0)
         {

